### ISSUE TITLE
Tooltip error in IE/Edge

### ISSUE THREAD

**Issue by martin-siden** [angular/material2#4069]
#### Bug, feature request, or proposal:
Hovering fast over multiple (3+) tooltips throws the following error in IE11 and Edge:

```
Unhandled Promise rejection: Unable to get property 'play' of undefined or null reference ; Zone: <root> ; Task: Promise.then ; Value: TypeError: Unable to get property 'play' of undefined or null reference TypeError: Unable to get property 'play' of undefined or null reference
```

#### What is the expected behavior?
I would like to be able to hover fast over as many tooltip items as i'd like without getting any error. 

#### What is the current behavior?
Hovering over one or two tooltip elements fast works, but 3+ (dragging your mouse pointer fast over them) throws the error. Everhything works as expected in Chrome/FF but not IE/Edge.

#### What are the steps to reproduce?
Have a look at this Plunkr:
http://plnkr.co/edit/cQ0mGZM41iaccV5ZOIdt?p=preview

#### What is the use-case or motivation for changing an existing behavior?
Better support for IE11 and Edge.

#### Which versions of Angular, Material, OS, browsers are affected?

- Angular v4.0.2
- Material v2.0.0-beta.3
- Windows 10
- Internet Explorer 11
- Edge

#### Is there anything else we should know?
My real app is built using Angular CLI and I have enabled the following polyfills:

```
/** IE9, IE10 and IE11 requires all of the following polyfills. **/
import 'core-js/es6/symbol';
import 'core-js/es6/object';
import 'core-js/es6/function';
import 'core-js/es6/parse-int';
import 'core-js/es6/parse-float';
import 'core-js/es6/number';
import 'core-js/es6/math';
import 'core-js/es6/string';
import 'core-js/es6/date';
import 'core-js/es6/array';
import 'core-js/es6/regexp';
import 'core-js/es6/map';
import 'core-js/es6/set';

/** IE10 and IE11 requires the following for NgClass support on SVG elements */
import 'classlist.js';  // Run `npm install --save classlist.js`.

/** IE10 and IE11 requires the following to support `@angular/animation`. */
import 'web-animations-js';  // Run `npm install --save web-animations-js`.

/** Evergreen browsers require these. **/
import 'core-js/es6/reflect';
import 'core-js/es7/reflect';
```

**Comment by HopScotch47** [comment]
I get the same error message randomly on IE11 and Edge maybe 1 out of 3 times when I open a dialog component.

**Comment by martin-siden** [comment]
Dialogs works fine for me in both IE and Edge.

**Comment by martin-siden** [comment]
My issue was solved by updating `web-animations-js` to `v2.2.4`.
